fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBMetaDataStore.java

```diff
@@ -266,8 +310,12 @@ protected RecordMetaData buildMetaData(RecordMetaDataProto.MetaData metaDataProt
         return instrument(FDBStoreTimer.Events.SAVE_META_DATA, future);
     }
 
-    // TODO: FDBMetadataStore should assign index keys based on a counter (https://github.com/FoundationDB/fdb-record-layer/issues/11)
     public CompletableFuture<RecordMetaData> getRecordMetaDataAsync(boolean errorIfMissing) {
+        return getRecordMetaDataAsync(null, errorIfMissing);
+    }
+
+    // TODO: FDBMetadataStore should assign index keys based on a counter (https://github.com/FoundationDB/fdb-record-layer/issues/11)
+    public CompletableFuture<RecordMetaData> getRecordMetaDataAsync(@Nullable Descriptors.FileDescriptor localMetaDataFileDescriptor, boolean errorIfMissing) {
```

Comment:
This seems like it also should but the `localMetaDataFileDescriptor` at the end (and it should use the same parameter name as whatever is decided for the other methods).

Reply:
sure.